msg/orbit_status.msg

```diff
@@ -1,6 +1,14 @@
+# ORBIT_YAW_BEHAVIOUR
```

Comment:
That's not also what I meant, sorry. I meant specifically before the `yaw_behaviour` field.

Reply:
I followed the same style as in other files like https://github.com/PX4/Firmware/blob/master/msg/telemetry_status.msg

It seems to be a bit messy in the msg files.

Reply:
> I followed the same style as in other files like https://github.com/PX4/Firmware/blob/master/msg/telemetry_status.msg

I don't agree with that style either. Anyway, I guess it's acceptable then. We have to eventually try to template the uORB msgs format.